cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+Cypress.Commands.add('fillNewEmployeeForm',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="city"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkDataDoesntExist',
+  (fullName, salary) => {
+    cy.get('tbody').contains(fullName).should('not.exist');
+    cy.get('tbody').contains(salary).should('not.exist');
+  });
+
+Cypress.Commands.add('hasData', (inputName, compareValue) => {
+  cy.contains(inputName).parent().should(($column) => {
+    const array = [...$column].map((td) => td.innerText.split('\t'));
```

Comment:
Please use a variable name that we can easily understand. `rowTextItems` or smth like that

Reply:
what do you mean? what I should use it for?